sw/device/silicon_creator/lib/cert/tpm_cik.hjson

```diff
@@ -53,7 +53,7 @@
         not_before: "20230101000000Z",
         not_after: "20500101000000Z",
         subject: [
-            { country: "USA" },
+            { country: "US" },
```

Comment:
@vbendeb: FYI

Reply:
yes, this is a good catch (would have been highligted by openssl verify ;) )